site2/docs/cookbooks-tiered-storage.md

```diff
@@ -30,6 +33,14 @@ Pulsar uses multi-part objects to upload the segment data. It is possible that a
 We recommend you add a life cycle rule your bucket to expire incomplete multi-part upload after a day or two to avoid
 getting charged for incomplete uploads.
 
+## The Filesystem offloading mechanism
+
+A topic data was stored with a lot of ledgers. Offload these ledgers into filesystem, we need a path for it.
+The implementation of the Filesystem offload to generate the path for **tenant/namespace/topic/ledgerId**.
```

Comment:
The path of the data is `basePath` +  /tenant/namespace/domain/topic/ +  ledgerId-UUID right?

Reply:
yes, it should be in this way.